forLim.py

```diff
@@ -227,13 +224,21 @@ def initGui(self):
         QObject.connect(self.dlg.LE_polylines, SIGNAL("editingFinished()"), self.check_polylines_path)
         
         
+        ## Select Method Menu
+        #Set convolution as default
+        self.dlg.radio_TR.setChecked(True)
```

Comment:
Sets Triangulation method as default for dev purposes can be switched in the future when production state is reached.